src/Compiler/Utilities/FileSystem.fs

```diff
@@ -530,7 +530,7 @@ type DefaultFileSystem() as this =
 
         // We want to use mmaped files only when:
         //   -  Opening large binary files (no need to use for source or resource files really)
-        //   -  Running on mono, since its MemoryMappedFile implementation throws when "mapName" is not provided (is null).
+        //   -  Not running on mono, since its MemoryMappedFile implementation throws when "mapName" is not provided (is null).
```

Comment:
We don't do mono checks any longer the comment can go by by.

Reply:
Also we don't do shadowcopy on coreclr, we probably need to think about what to do about that.